servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/KeepAliveManager.java

```diff
@@ -129,7 +129,10 @@ protected void channelIdle(final ChannelHandlerContext ctx, final IdleStateEvent
             final long idlenessThresholdNanos = keepAlivePolicy.idleDuration().toNanos();
             pingWriteCompletionListener = idlenessThresholdNanos > 0 ? future -> {
                 assert channel.eventLoop().inEventLoop();
-                if (future.isSuccess() && keepAliveState == State.KEEP_ALIVE_ACK_PENDING) {
+                if (!future.isSuccess()) {
+                    LOGGER.debug("{} Failed writing a PING frame after idleness is detected", channel, future.cause());
```

Comment:
Consider mentioning in the log that the cannel is now going to be proactively closed.